documentation/api/query/v4/operators.markdown

```diff
@@ -204,6 +204,38 @@ When querying a particular endpoint, such as `/pdb/query/v4/nodes` the endpoint
 the context for the query. While querying the [root] endpoint requires specifying a
 context explicitly.
 
+## Paging Operators (`limit`, `offset`, `order_by`)
+
+PuppetDB allows specification of paging clauses within a "from" clause in a
+query or subquery. The `limit` and `offset` operators both accept an
+integer-valued argument, and `order_by` accepts a vector of either column names
+or vector pairs containing a column name and an ordering of "asc" or "desc".
+For example,
+
+    ["order_by", ["certname", ["timestamp", "desc"]]]
+
+uses both possibilities. When no ordering is explicitly specified, as in the
+case of "certname" in the example above, ascending order is assumed. Here are a
+few examples of queries using paging operators:
+
+Return the most recent ten reports for a certname:
+
+    ["from", "reports", ["=", "certname", "myserver"], ["order_by", [["timestamp", "desc"]]], ["limit", 10]]
+
+Return the next page of ten reports:
+
+    ["from", "reports", ["=", "certname", "myserver"], ["order_by", [["timestamp", "desc"]]], ["limit", 10], ["offset", 10]]
+
+Return the most recent ten reports for any certname:
+
+    ["from", "reports", ["order_by", [["timestamp", "desc"]]], ["limit", 10]]
+
+Return the nodes represented in the ten most recent reports:
+
+    ["from", "nodes", ["in", "certname", ["from", "reports", ["extract", "certname"], ["limit", 10], ["order_by", [["certname", "desc"]]]]]]
```

Comment:
Examples don't wrap in the docs website, maybe these few examples would read better if they were multi-line?